components/brave_new_tab_ui/containers/newTab/gridTile.tsx

```diff
@@ -37,13 +36,9 @@ interface Props {
 }
 
 class TopSite extends React.PureComponent<Props, {}> {
-  onTogglePinnedTopSite (site: NewTab.Site) {
-    this.props.actions.toggleGridSitePinned(site)
-  }
-
   onIgnoredTopSite (site: NewTab.Site) {
-    this.props.actions.removeGridSite(site)
-    this.props.actions.showGridSiteRemovedNotification(true)
+    deleteMostVisitedTile(site.url)
+    this.props.actions.showTilesRemovedNotice(true)
```

Comment:
Similarly, consider moving this API call to the redux handler (which should be called after the action something like `removeTile(tile)` instead of the effect of showing the notice).

Reply:
done!